src/main/steps/applicant1/hub-page/joint/jointTemplateSelector.ts

```diff
@@ -21,11 +23,16 @@ export const getJointHubTemplate = (
     case State.AwaitingClarification:
       return '/awaiting-clarification.njk';
     case State.ClarificationSubmitted:
-      return '/clarification-submitted.njk';
+      if (userCase.isAdminClarificationSubmitted === YesOrNo.YES) {
+        return '/awaiting-legal-advisor-referral.njk';
+      } else {
+        return '/clarification-submitted.njk';
+      }
     case State.AwaitingAmendedApplication:
       return '/awaiting-amended-application.njk';
     case State.ConditionalOrderPending:
       return '/conditional-order-pending.njk';
+    case State.AwaitingAdminClarification:
```

Comment:
should we not show '/awaiting-clarification.njk' for this state?

Reply:
No, AC1 on the ticket is to change this to awaiting-legal-advisor-referral.